Corona-Warn-App/src/main/java/de/rki/coronawarnapp/storage/TracingRepository.kt

```diff
@@ -91,29 +93,17 @@ class TracingRepository @Inject constructor(
         it.taskState.isActive && it.taskState.request.type == RiskLevelTask::class
     }
 
-    /**
-     * Refresh the diagnosis keys. For that isRefreshing is set to true which is displayed in the ui.
-     * Afterwards the RetrieveDiagnosisKeysTransaction and the RiskLevelTransaction are started.
-     * Regardless of whether the transactions where successful or not the
-     * lastTimeDiagnosisKeysFetchedDate is updated. But the the value will only be updated after a
-     * successful go through from the RetrievelDiagnosisKeysTransaction.
-     */
-    fun refreshDiagnosisKeys() {
-        scope.launch {
-            taskController.submitBlocking(
-                DefaultTaskRequest(
-                    DownloadDiagnosisKeysTask::class,
-                    DownloadDiagnosisKeysTask.Arguments(),
-                    originTag = "TracingRepository.refreshDiagnosisKeys()"
-                )
-            )
-            taskController.submit(
-                DefaultTaskRequest(
-                    RiskLevelTask::class,
-                    originTag = "TracingRepository.refreshDiagnosisKeys()"
-                )
+    fun refreshRiskResult() = scope.launch {
+        Timber.tag(TAG).d("refreshRiskResults()")
+
+        riskWorkScheduler.runRiskTasksNow()
+
+        taskController.submit(
+            DefaultTaskRequest(
+                RiskLevelTask::class,
+                originTag = "TracingRepository.refreshDiagnosisKeys()"
```

Comment:
Should the tag be updated as well?

Reply:
Yes, currently only affects logging though.

Reply:
Done.